flink-core/src/main/java/org/apache/flink/configuration/Configuration.java

```diff
@@ -595,6 +595,35 @@ public boolean containsKey(String key){
 		}
 	}
 
+	/**
+	 * Checks whether there is an entry for the given config option
+	 *
+	 * @param configOption The configuration option
+	 *
+	 * @return <tt>true</tt> if a valid (current of deprecated) key of the config option is stored,
+	 * <tt>false</tt> otherwise
+	 */
+	public boolean contains(ConfigOption<?> configOption) {
```

Comment:
Could you annotate this with `@PublicEvolving` similar to other methods using `ConfigOption`s.